pymeasure/instruments/fwbell/fwbell5080.py

```diff
@@ -24,9 +24,10 @@
 
 from pymeasure.instruments import Instrument
 from pymeasure.instruments.validators import strict_discrete_set
-from numpy import array, float64
 from time import sleep
```

Comment:
As you are restructuring the imports, you could move the time import to the top (standard library).